ui/plugins/eu.esdihumboldt.hale.ui.functions.numeric/src/eu/esdihumboldt/hale/ui/functions/numeric/SequentialIDParameterPage.java

```diff
@@ -298,6 +303,28 @@ public void modifyText(ModifyEvent e) {
 	 */
 	protected void updateStatus() {
 		boolean complete = true;
+		setErrorMessage(null);
+
+		try {
+
+			Integer.parseInt(startValue.getText());
+
+			if (startValue.getText().charAt(0) == '0') {
```

Comment:
This only checks the first character in the string.

This very is different from the check that is used in the function implementation. It would probably make sense to use the same condition in both places so it is consistent?

Reply:
I believe that in both places I am checking that the generated ID (and thus the ID example) does not start with 0.

In the function in fact we check if startValue==0:

```
id = Integer.parseInt(startValue);
if (id == 0) {
		throw new TransformationException(
		"ERROR: the starting value for ID creation should be larger than 0");
		}
```

Reply:
Still the fact remains that the checks are very different, with different results for different input strings (e.g. if someone uses leading zeros in the input string).